scripts/ioc_deploy.py

```diff
@@ -1,31 +1,39 @@
 #!/usr/bin/python3
 """
 ioc-deploy is a script for building and deploying ioc tags from github.
-It will create a shallow clone of your IOC in the standard release area at the correct path and "make" it.
-If the tag directory already exists, the script will exit.
+
+It will create a shallow clone of your IOC in the standard release area at the
+correct path and "make" it. If the tag directory already exists, the script
+will exit.
```

Comment:
Nit: Maybe this should be updated a bit or an additional paragraph added since now we have an alternate path for just modifying permissions.

Reply:
This is fair, I can update the readme more for the second path including a new example.